packages/components/src/popover/index.js

```diff
@@ -80,24 +81,21 @@ class Popover extends Component {
 	}
 
 	componentWillUnmount() {
-		this.toggleWindowEvents( false );
-
 		clearTimeout( this.focusTimeout );
+		this.toggleAutoRefresh();
 	}
 
-	toggleWindowEvents( isListening ) {
-		const handler = isListening ? 'addEventListener' : 'removeEventListener';
-
-		window.cancelAnimationFrame( this.rafHandle );
-		window[ handler ]( 'resize', this.throttledComputePopoverPosition );
-		window[ handler ]( 'scroll', this.throttledComputePopoverPosition, true );
-	}
-
-	throttledComputePopoverPosition( event ) {
-		if ( event.type === 'scroll' && this.contentNode.current.contains( event.target ) ) {
-			return;
+	toggleAutoRefresh() {
+		if ( this.autoRefresh ) {
+			clearInterval( this.autoRefresh );
+			this.autoRefresh = null;
+		} else {
+			// Checks every 0.2s if the popover needs to be resized or repositioned
+			this.autoRefresh = setInterval( () => {
+				window.cancelAnimationFrame( this.rafHandle );
+				this.rafHandle = window.requestAnimationFrame( () => this.refresh() );
```

Comment:
I think we should store the last position of the anchor element and the last size of the popover content and compare with the previous value before calling refresh :)

I'm having a weird behavior where the scrollbars appear/disappear continuously and I suspect that this change will fix it :)

![crazy-popover](https://user-images.githubusercontent.com/272444/47911185-11819b00-de95-11e8-90b6-e9fbda60cd74.gif)

Reply:
Good catch! 

We were already checking that the size and position didn't change in the `updatePopoverSize` and `computePopoverPosition` methods. 

The problem was on how the size was being calculated, as it was not having in consideration the actual height of the content, just the visible the part.

I fixed it by checking the `scrollHeight` property instead of using `getBoundingClientRect().height`.